python/setup.py

```diff
@@ -66,7 +66,8 @@
         language='c++',
         undef_macros=["NDEBUG"],
         include_dirs=["pybind11/include"],
-        extra_compile_args=['-std=c++17']
+        extra_compile_args=['-std=c++17', '-fopenmp'],
+        extra_link_args=['-glomp']
```

Comment:
Did you mean `-lgomp` (reverse order of the `l` and `g` letters) here?

Reply:
Thanks I don't know - this is just something I found on the internet. On my machine "everything" just  seems to work.

Reply:
drop the gomp link - don't want to limit to gcc.